app/controllers/devise/two_factor_authentication_controller.rb

```diff
@@ -53,12 +53,6 @@ def check_already_authenticated
       redirect_to profile_path if user_fully_authenticated?
     end
 
-    def use_totp?
-      # Present the TOTP entry screen to users who are TOTP enabled,
-      # unless the user explictly selects SMS or voice
-      current_user.totp_enabled? && !use_sms_or_voice_otp_delivery?
-    end
```

Comment:
I removed this because a TOTP-enabled user can never select any other option before they land on the `show` view, which is the only place this method was called. When a TOTP-enabled user signs in, they are automatically asked for their Authenticator app code, with the option to use SMS or voice instead. If they choose one of those options, they go to the `otp/send` path, which never calls this `use_totp?` method.